### ISSUE TITLE
The Install script deletes the repository if it exists in ~/.themes

### ISSUE THREAD

**Issue by jwhendy** [vinceliuice/Abrus-gtk-theme#3]
I found your incredible stash of themes via this [reddit post](https://www.reddit.com/r/unixporn/comments/7x90xa/there_doesnt_seem_to_be_a_lot_of_good_gtk_themes/du80rgb/). During the install, I was getting loads of errors like this on Abrus, Azure, and Yosemite, as well as finding that the cloned directory was gone, too!
```
Abrus-gtk-theme]$ ./Install 
   Installing Abrus ...
./Install: line 43: cd: /home/jwhendy/.themes/Abrus-gtk-theme/src/gnome-shell: No such file or directory
cp: cannot stat 'gnome-shell.css': No such file or directory
cp: cannot stat 'extensions': No such file or directory
cp: cannot stat 'pad-osd.css': No such file or directory
cp: cannot stat 'message-indicator-symbolic.svg': No such file or directory
...
```

It turns out, this is because I cloned to `~/.themes/foo` and the `Install` script deletes all wildcard completions of the theme name!
```
rm -rf $DEST_DIR/Abrus*
```

Interestingly, this doesn't happen on `stylish-gtk-theme`. I thought it was due to case sensitivity (and the repo is `stylish-gtk-theme` while the theme names are `Styllish-*`, but your script there is actually much better constructed. You iterate through each specific theme directory vs. `ThemeName*` completions:
```
local THEME_DIR=${dest}/${name}${variant}${size}${color}
[[ -d ${THEME_DIR} ]] && rm -rf ${THEME_DIR}
```

Indeed, this is no problem:
```
[.themes]$ mv stylish-gtk-theme/ Stylish-gtk-theme
[.themes]$ cd Stylish-gtk-theme/
[Stylish-gtk-theme]$ ./Install 
Installing '/home/jwhendy/.themes/Stylish'...
Installing '/home/jwhendy/.themes/Stylish-Tang'...
...
```
This also seems better as why wildcard delete stuff on a user's system you might not have put there? The approach with `stylish` only deletes what you know you created or conflicts.

---

*tl;dr*
Long story short (or made longer!), I don't really want to make a special directory just for themes, so I want everything in `~/.themes` since it already exists for that purpose. Thus, I'd prefer if scripts didn't delete the src directory and just ignored it. This way, I can:
```
cd ~/.theme/name
git fetch && git merge
./Install
```
It looks like your approach to `stylish` would do nicely for the rest of the themes in your ecosystem.

**Comment by vinceliuice** [comment]
I'll fix this
